src/schema/v2/partner/partner.ts

```diff
@@ -646,7 +646,7 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
           )
 
           const gravityArgs = {
-            published: true,
+            private: false,
```

Comment:
Is this an intentional change here?

Reply:
oh yeah I meant to leave a comment about that, it was a drive-by - `published` is [not actually a parameter on this Gravity endpoint](https://github.com/artsy/gravity/blob/764e9360a762896c0203477b212f783bab09f5aa/app/api/v1/partners_locations_endpoint.rb#L70), but `private` is.

But looking at this again, I should add a spec for it (and weirdly I think this should be giving a type error due to [this declaration](https://github.com/artsy/metaphysics/blob/3f4b5272944121443f3cc1559acd72ef6faa1895/src/lib/loaders/loaders_without_authentication/gravity.ts#L156), but it doesn't seem to be).

I'll update quick so this is an actual change and not half-assed!

Reply:
Nice catch on the parameters